### PR TITLE
changes for qcell status

### PR DESCRIPTION
**Pull request by SnehaCRao**

### CODE REVIEW

File: SamplifyAPIClient/Response.cs [hunk 1]
```diff
@@ -127,6 +127,13 @@ public class DetailedLineItemReportResponse : Response
         public DetailedLineItemReport Data { get; set; }
     }
 
+    [DataContract]
+    public class QuotaCellReponse : Response
+    {
+        [DataMember(Name = "data")]
+        public DetailedLineItemReport Data { get; set; }
```
Comment (ashajyothi828) [inline SamplifyAPIClient/Response.cs]: Should this contain report in the response? dont think so

**Review by ashajyothi828 — CHANGES_REQUESTED** [review]
The response type needs to be fixed I think. Also I dont see any changes for returning the quota cell state in the lineitem level. Dont we return that?

File: SamplifyAPIClient/Response.cs (hunk 1 above)
Comment (SnehaCRao) [reply SamplifyAPIClient/Response.cs]: Have updated it, thanks for pointing it out.

**Comment by SnehaCRao** [comment]
> The response type needs to be fixed I think. Also I dont see any changes for returning the quota cell state in the lineitem level. Dont we return that?

The response type is fixed. The status change has been added in LineItem.cs file.

**Review by ashajyothi828 — APPROVED** [review]

**Pull request merged**